src/properties.jl

```diff
@@ -496,6 +497,11 @@ function class_getproperty(::Type{DatasetCreateProperties}, p::Properties, name:
     name === :external    ? API.h5p_get_external(p) :
     name === :filters     ? get_filters(p) :
     name === :layout      ? get_layout(p) :
+    name === :no_attrs_hint ?
```

Comment:
add to the docstring for `DatasetCreateProperties`

Reply:
done in [a523356](https://github.com/JuliaIO/HDF5.jl/pull/944/commits/a5233563d8fd83c1e0476b5470365f374ea0e3b4)